### PR TITLE
Implemented an overall progress bar for the store cmd

### PR DESCRIPTION
**Pull request by glaxx**

### CODE REVIEW

File: snapshot.go [hunk 1]
```diff
@@ -9,12 +9,12 @@ package knoxite
 
 import (
 	"encoding/json"
+	uuid "github.com/nu7hatch/gouuid"
```
Comment (muesli) [inline snapshot.go]: I prefer having a separate block of external imports below, as it used to be. No dealbreaker though.

Comment (glaxx) [inline snapshot.go]: Sorry, this was unintentional. *cough* go fmt *cough*

Comment (muesli) [inline snapshot.go]: Note: If you keep the empty line between the regular imports and the external ones, go fmt will treat and sort it as a separate block.

Comment (glaxx) [inline snapshot.go]: Fixed d9067f2

**Pull request merged**